### PR TITLE
Header and basic yaml

### PR DESCRIPTION
**Pull request by mwernimont**
@jiwalker-usgs Header is in place and basic template finding setup in the yaml

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -45,3 +45,48 @@ fetch:
 process:
 visualize:
 publish:
+  -
+    id: headerCSS
```
Comment (jiwalker-usgs) [inline viz.yaml]: I don't think this is needed, the css needs to be added to resources, but the file is coming from vizlab

Comment (mwernimont) [inline viz.yaml]: I wasn't getting any of its CSS until I added it to the viz.yaml though...thoughts?

Comment (mwernimont) [inline viz.yaml]: oh I get it, will do

**Comment by mwernimont** [comment]
@jiwalker-usgs Better?

**Pull request merged**